core/modules/simpletest/backdrop_web_test_case.php

```diff
@@ -520,10 +520,10 @@ protected function fail($message = NULL, $group = 'Other') {
    *   The type of assertion - examples are "Browser", "PHP".
    * @param $caller
    *   The caller of the error.
-   * @return
-   *   FALSE.
+   * @return false
```

Comment:
Shouldn't this remain uppercase?

Reply:
We're not consistent on it. Up until recently I was using `FALSE` and `NULL` in phpDoc but @indigoxela pointed out lowercase is more common.

Reply:
This inconsistency was discovered as part of https://github.com/backdrop/backdrop/pull/4915.

Reply:
I think we decided to standardize on lowercase.